create fist PoC for this project (#1)
packit-service/requre: README.md

> # Record Requests!

Reviewer: This does not match the `requre` shortcut...;)
```suggestion
# REQUest REcording
```

Requre \[rekure\] 

Is Library for storing output of various function and methods to 

> persistenst storage and able to replay the stored output to functions

Reviewer: ```suggestion
persistent storage and be able to replay the stored output to functions
```

back

## Plan and current status

>  - Used for testing packit-service organization projects

Reviewer: ```suggestion
 - Used for testing [packit-service](https://github.com/packit-service) organization projects
```
